docker/appserver/WebSphere/src/scripts/configureJdbc.py

```diff
@@ -41,18 +45,11 @@ def createDatasource(datasourceName, providerName, authAlias, properties):
 	datasourceId = AdminJDBC.createDataSource(node, server, providerName, datasourceName, attributes)
 	return(datasourceId)
 
-
-#p0 = AdminConfig.listTemplates('JDBCProvider')
-#print 'JDBCProvider templates: ', p0
-#p1 = AdminConfig.listTemplates('DataSource')
-#print 'DataSource templates: ', p1
-
-
-
-
 createTemplatedProvider('Oracle JDBC Driver (XA)', 		 'oracle.jdbc.xa.client.OracleXADataSource', 		'/work/drivers/ojdbc${oracle.driver.jdkversion}.jar')
 createTemplatedProvider('Microsoft SQL Server JDBC Driver (XA)', 'com.microsoft.sqlserver.jdbc.SQLServerXADataSource',  '/work/drivers/mssql-jdbc.jar')
 createProvider('H2 JDBC Driver (XA)', 'org.h2.jdbcx.JdbcDataSource', 'classpath=/work/drivers/h2.jar,xa=true')
+createProvider('MySQL JDBC Driver', 'com.mysql.cj.jdbc.MysqlXADataSource', 'classpath=/work/drivers/mysql-connector-java.jar')
+createProvider('MariaDB JDBC Driver', 'com.mysql.cj.jdbc.MysqlXADataSource', 'classpath=/work/drivers/mysql-connector-java.jar')
```

Comment:
Je hoeft deze MariaDB JDBC Driver niet op te nemen, omdat het dezelfde is als de MySQL driver